railties/lib/rails/generators/app_base.rb

```diff
@@ -193,11 +196,29 @@ def webserver_gemfile_entry # :doc:
       end
 
       def include_all_railties? # :doc:
-        options.values_at(:skip_active_record, :skip_action_mailer, :skip_test, :skip_sprockets, :skip_action_cable).none?
+        [
+          options.values_at(
+            :skip_active_record,
+            :skip_action_mailer,
+            :skip_test,
+            :skip_sprockets,
+            :skip_action_cable
+          ),
+          skip_active_storage?
```

Comment:
I think that specifying `skip_active_record` directly is more simpler.

Reply:
Yeah it is simpler, but I think it is more correctly use `skip_active_storage?`